roles/server/tasks/main.yml

```
@@ -43,6 +51,21 @@
     - local-json
     - config
 
+- name: Check if "welcome-directs-{{ freefeed_env }}.yml" exists
+  stat:
+    path: "{{ role_path }}/files/welcome-directs-{{ freefeed_env }}.yml"
+  register: welcome_directs_file
+
+- name: Copy "welcome-directs-{{ freefeed_env }}.yml" to welcome-directs if exists
+  become: yes
+  copy:
+    src: "welcome-directs-{{ freefeed_env }}.yml"
```

@davidmz even better, we can use the same variable, just different content for different environments. And only use `freefeed_env` in target config file name.
```suggestion
    content: "{{ welcome-directs | to_nice_yaml }}"
```